### PR TITLE
add prerelease lifecycle tags to scheduling.k8s.io

### PR DESCRIPTION
**Pull request by deads2k**
This adds the lifecycle annotation tags matching https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/1635-prevent-permabeta/README.md 

The clock is set for removal of beta levels in 1.22.  The APIs were deprecated in 1.14

/kind cleanup
/priority important-soon
@kubernetes/sig-scheduling-api-reviews 

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/api/scheduling/v1beta1/types.go [hunk 1]
```diff
@@ -24,6 +24,10 @@ import (
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.11
+// +k8s:prerelease-lifecycle-gen:deprecated=1.14
+// +k8s:prerelease-lifecycle-gen:removed=1.22
```
Comment (Huang-Wei) [inline staging/src/k8s.io/api/scheduling/v1beta1/types.go:29]: One qq: what should scheduling team do in 1.22? Remove the v1beta1 Priority API in the code, and then run hack/xyz to update the tags? Or something else.

**Review by Huang-Wei — COMMENTED** [review]
/lgtm
/approve

Thanks for driving this. One question below.

File: staging/src/k8s.io/api/scheduling/v1beta1/types.go (hunk 1 above)
Comment (deads2k) [reply staging/src/k8s.io/api/scheduling/v1beta1/types.go:29]: > One qq: what should scheduling team do in 1.22? Remove the v1beta1 Priority API in the code, and then run hack/xyz to update the tags? Or something else.

All clients and controllers should be updated to use the v1 level of the API.

In 1.22, the REST handler code will automatically stop serving APIs based on these tags, so no other action should be required.

Comment (Huang-Wei) [reply staging/src/k8s.io/api/scheduling/v1beta1/types.go:29]: SG. Thanks for explanation.

**Pull request merged**